autogpt_platform/backend/backend/blocks/__init__.py

```diff
@@ -76,6 +77,6 @@ def all_subclasses(clz):
     if block.disabled:
         continue
 
-    AVAILABLE_BLOCKS[block.id] = block
+    AVAILABLE_BLOCKS[block.id] = cls
```

Comment:
Any changes other than the one in llm.py, is simply propagating this change:
Making this global variable holding type classes instead of object.